types/openlayers/index.d.ts

```diff
@@ -12383,11 +12384,11 @@ declare module ol {
          * The size is the pixel dimensions of the box into which the calculated extent
          * should fit. In most cases you want to get the extent of the entire map,
          * that is `map.getSize()`.
-         * @param {ol.Size} size Box pixel size.
+         * @param {ol.Size=} size Box pixel size.
```

Comment:
This seems like a good reason to follow the `no-redundant-jsdoc` lint rule so you don't have to update the same thing in two different places.